.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Release 
+on:
+    workflow_dispatch:
+    
+jobs:
+
+    package_app:
+        name: Package and publish
+        env:
+            GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
Will sub this out for the GITHUB_TOKEN , I was afraid the perms wouldn't be enough to create a release